index.html

```diff
@@ -134,8 +139,18 @@
           $("#jsonInput").val(JSON.stringify(testDataClinical, null, '    '));
         }
         $("#parseJson").on("click", function() {
-          //initialise with advanced view
-          initViews(true);
+           if (viewType === "simple") {
+            initViews(false);
+            if (location.href.indexOf("?view=") === -1) {  
+              window.history.pushState('simpleView', '', location.href+="?view=simple");
```

Comment:
To enable the correct functionality of back/forward buttons I'll need to refresh the page instead of history.pushState and this will mean loss of data. That is if you change the data in the textBox in advancedView and then change to simpleView, the change is not reflected.